src/components/Common/Auth/ConnectWalletButton.tsx

```diff
@@ -47,6 +48,8 @@ const ConnectWalletButton = ({ handleSign, signing }: Props) => {
 
   const { mounted } = useIsMounted()
 
+  const { openConnectModal } = useConnectModal()
+
```

Comment:
we can safely get rid of <Modal> in this component and related unused states and functions 🙇🏻

Reply:
Should be all cleaned up now 🙂